rmoss_util/include/rmoss_util/url_resolver.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 RoboMaster-OSS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This program is based on https://github.com/ros-perception/image_common
+// which are released under the BSD License.
+// https://opensource.org/license/bsd-3-clause/
+
+#ifndef RMOSS_UTIL__URL_RESOLVER_HPP_
+#define RMOSS_UTIL__URL_RESOLVER_HPP_
+
+#include <string>
+
+namespace rmoss_util
+{
+class URLResolver
+{
+public:
+  /**
+   * @brief
+   *
+   * @param url
+   * @return std::string
+   */
+  static std::string resolveURL(const std::string & url);
```

Comment:
代码风格建议统一，函数命名尽可能采用下划线方式，而不是驼峰方式，其他地方也是。
```suggestion
  static std::string resolve_url(const std::string & url);
```